build/e2e_test.go

```diff
@@ -196,55 +196,131 @@ func retryTest(t *testing.T, f func(fatalf func(format string, args ...interface
 }
 
 func TestLargeFileOption(t *testing.T) {
```

Comment:
minor: do we really need to add so many e2e tests? These are slow to run. I would suspect you would get better mileage out of just unit testing IgnoreSizeMax directly.

Reply:
Updated builder_test.go with unit tests and reverted e2e test (except for one line minor change)